generator/lib/control.ml

```diff
@@ -207,6 +210,18 @@ module Volume(R: RPC) = struct
        "extra metadata associated by [set] is not copied."]
       (dbg @-> sr @-> key @-> returning volume errors)
 
+  let copy = R.declare "copy"
+      ["[copy sr volume dest_sr] creates a new volume as a writeable copy of ";
+       "[volume] in [new_sr]. [new_sr] may be the same as [sr] and the operation ";
```

Comment:
I think we should refer to this consistently. Either `new_sr` or `dest_sr`, but not both.

Reply:
yes, good point.